File: backend/src/main/java/jp/co/iwaki/data/api/domain/model/resource/market/MarketTransactionResource.java
package jp.co.iwaki.data.api.domain.model.resource.market;

import lombok.Data;

/**
 * 品目毎の取扱高
 */
@Data
public class MarketTransactionResource {

    /** 年度 */
    private String year = "";
    /** 野菜 */
    private MarketItemResource vegetable = new MarketItemResource();
    /** 果実 */
    private MarketItemResource fruit = new MarketItemResource();
    /** 鳥卵加工品 */
    private MarketItemResource processedBirdEgg = new MarketItemResource();
    /** 青果部合計 */
    private MarketItemResource fruitAndVegetableSectionTotal = new MarketItemResource();
    /** 鮮魚 */
    private MarketItemResource freshFish = new MarketItemResource();
    /** 冷凍 */
    private MarketItemResource frozen = new MarketItemResource();
    /** 塩干加工品 */
    private MarketItemResource saltDriedProducts = new MarketItemResource();
    /** 水産物部合計 */
    private MarketItemResource fisheriesDepartmentTotal = new MarketItemResource();
    /** 花き部 */
    private MarketItemResource flowerPart = new MarketItemResource();
    /** 数量総計 */
    private Long totalQuantity = 0L;
    /** 金額総計 */
    private Long totalAmount = 0L;
}

File: backend/src/main/kotlin/iwakiDataApi/resource/statistics/market/MarketTransactionResource.kt
package iwakiDataApi.resource.statistics.market

import iwakiDataApi.resource.statistics.market.MarketItemResource

data class MarketTransactionResource(
    val year: String = "",
    // 野菜
    val vegetable: MarketItemResource,
    // 果実
    val fruit: MarketItemResource,
    // 鳥卵加工品
    val processedBirdEgg: MarketItemResource,
    // 青果部合計
    val fruitAndVegetableSectionTotal: MarketItemResource,
    // 鮮魚
    val freshFish: MarketItemResource,
    // 冷凍
    val frozen: MarketItemResource,
    // 塩干加工品
    val saltDriedProducts: MarketItemResource,
    // 水産物部合計
    val fisheriesDepartmentTotal: MarketItemResource,
    // 花き部
    val flowerPart: MarketItemResource,
    // 数量総計
    val totalQuantity: Long = 0,
    // 金額総計
    val totalAmount: Long = 0
)
